app/services/toolbars/repository_rows_service.rb

```diff
@@ -130,6 +130,7 @@ def export_consumption_action
 
     def export_actions
       {
+        name: 'export',
```

Comment:
it's a group, should not have an individual name

Reply:
changed to --> "export_group"